### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by razmikmelikbekyan**
*Issue #, if available:*

*Description of changes:*
Basic PEP 8 corrections, 
Making code more readable,
Small fixes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: autogluon/model_zoo/model_zoo.py [hunk 1]
```diff
@@ -42,11 +43,12 @@ def get_model(name, **kwargs):
...
+
 def get_model_list():
     """Get the entire list of model names in model_zoo.
     Returns
     -------
     list of str
         Entire list of model names in model_zoo.
     """
-    return models.keys()
+    return list(models.keys()) + list(gcv.model_zoo.get_model_list())
```
Comment (Innixma) [inline autogluon/model_zoo/model_zoo.py]: @zhanghang1989 This appears to be change in functionality, confirm if correct.

File: autogluon/task/image_classification/image_classification.py [hunk 2]
```diff
@@ -1,43 +1,42 @@
...
     """
-    Classifier=Classifier
+    Classifier = Classifier
+
     @staticmethod
     def Dataset(*args, **kwargs):
         """Dataset for AutoGluon image classification tasks. 
            May either be a :class:`autogluon.task.image_classification.ImageFolderDataset`, :class:`autogluon.task.image_classification.RecordDataset`, 
-           or a popular dataset already built into AutoGluon ('mnist', 'cifar10', 'cifar100', 'imagenet').
+           or a popular dataset already built into AutoGluon ('mnist', 'fashionmnist', 'cifar10', 'cifar100', 'imagenet').
```
Comment (Innixma) [inline autogluon/task/image_classification/image_classification.py]: @zhanghang1989 confirm if this is correct

**Comment by szha** [comment]
Job PR-307-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-307/1/index.html

File: autogluon/model_zoo/model_zoo.py (hunk 1 above)
Comment (zhanghang1989) [reply autogluon/model_zoo/model_zoo.py]: Thanks @Innixma . This change is good to get gluoncv model_zoo.

File: autogluon/task/image_classification/image_classification.py (hunk 2 above)
Comment (zhanghang1989) [reply autogluon/task/image_classification/image_classification.py]: Thanks! This is correct.

**Pull request merged**